packages/showcase/src/user-controls/common-snippets/common-snippets.component.scss

```diff
@@ -35,3 +35,39 @@
     }
   }
 }
+
+// container-*
+.background-color-container {
+  $afterPaddingBlock: var(--mer-spacing--md);
+  $afterFontSize: var(--font-size-body-m);
+  $afterBlockSize: calc(($afterPaddingBlock * 2) + $afterFontSize);
+  position: relative;
+  padding: $afterBlockSize var(--mer-spacing--sm) var(--mer-spacing--sm)
+    var(--mer-spacing--sm);
+  border: 1px dashed var(--mer-border-color__on-surface);
+  border-radius: var(--mer-spacing--2xs);
+  &:after {
+    position: absolute;
+    inline-size: 100%;
+    text-align: center;
+    inset-inline-start: 0;
+    inset-block-start: 0;
+    font-size: var(--font-size-body-m);
+    line-height: 1;
+    padding: $afterPaddingBlock 12px;
+    font-style: italic;
+  }
+
+  &:after {
+    content: "Surface";
+  }
+  &.elevation-1:after {
+    content: "Elevation 1";
+  }
+  &.elevation-2:after {
+    content: "Elevation 2";
+  }
+  &.elevation-3:after {
+    content: "Elevation 3";
+  }
```

Comment:
Pseudo content must only be used for decorative purposes. Don't use it to render text content as it's not accessible.

https://accessibleweb.com/question-answer/how-is-css-pseudo-content-treated-by-screen-readers/

Another argument: This content type can't be selected with the mouse.

Reply:
@ncamera I will resolve this issue on a separate PR since it affects other showcase pages as well.